contracts/CcipResolver.sol

```diff
@@ -15,13 +15,18 @@ import {BytesLib} from "solidity-bytes-utils/contracts/BytesLib.sol";
  * Implements an ENS resolver that directs all queries to a CCIP read gateway.
  * Callers must implement EIP 3668 and ENSIP 10.
  */
+
 contract CcipResolver is IExtendedResolver, IMetadataResolver, SupportsInterface {
     using BytesUtils for bytes;
 
     struct CcipVerifier {
         string[] gatewayUrls;
         ICcipResponseVerifier verifierAddress;
     }
+    /**
+     * The idnetifier to store the default verifier
+     */
+    bytes32 private constant DEFAULT_VERIFIER = bytes32(0);
```

Comment:
I think it's probably better to call this the ROOT_NODE, as this is what we call it within the context on ENS